vignettes/images/function-flow.puml

```diff
@@ -0,0 +1,86 @@
+@startuml function-flow
+!theme cerulean-outline
+<style>
+action {
+    FontColor black
+}
+database {
+    FontColor black
+}
+.inclusion {
+    BackgroundColor lightblue
+}
+.exclusion {
+    BackgroundColor orange
+}
+</style>
+
+hide <<inclusion>> stereotype
+hide <<exclusion>> stereotype
+
+card classify_diabetes_status() as cd {
+    together {
+        database sssy
+        database sysi
+        database lpr_diag
+        database lpr_adm
+        database lmdb
+        database lab_forsker
+        database kontakter
+        database diagnoser
+        database bef
+    }
+
+    action "get_pregnancy_dates()" as pregnancy
+    action "get_pcos()" as pcos
+    action "get_diagnosis_date()" as diagnosis_date
+    action "join_lpr2" as lpr2
+    action "join_lpr3" as lpr3
```

Comment:
```suggestion
    action "join_lpr2()" as lpr2
    action "join_lpr3()" as lpr3
```

Always refer to functions with an ending `()` to differentiate it from a variable object.